element.go

```diff
@@ -44,6 +44,10 @@ func (e *Element) GetAttribute(k string) js.Value {
 	return e.v.Call("getAttribute", k)
 }
 
+func (e *Element) RemoveAttribute(k string) js.Value {
```

Comment:
```suggestion
func (e *Element) RemoveAttribute(k string) {
```
It doesn't return a value, I believe.